recipes/dca/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "DCA" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f3b979c3e11de0dfba6503631db5e1abd5e926b9233ec956ab7da4badb31c4e6
+
+build:
+  skip: True # [not py36]
```

Comment:
This could also be a noarch package, isn't it? You can skip the python version on the pinning level.